Makefile

```diff
@@ -2,6 +2,7 @@ project = p88
 version ?= latest
 img = $(project):$(version)
 vol = $(shell pwd):/$(project)
+username = devops
```

Comment:
Adding this configuration, so the Ansible commands later know which folder to move the SSH keys information. Previously it was sending to the `root` folder and the container was breaking due to it.